### ISSUE TITLE
issue when importing eel on Python 3.8.5

### ISSUE THREAD

**Issue by felini69** [samuelhwilliams/Eel#400]
**Describe the problem**
I install last version of eel. write few line of code and start project. cant import eel. I have issue with importing eel. I have Python3.8.5 version on my Windows PC.

text from my CMD
import eel
ModuleNotFoundError: No module named 'eel'

**Code snippet(s)**
![image](https://user-images.githubusercontent.com/73040536/96353596-8845ec00-10de-11eb-8b30-b7bb71b2cf01.png)

- [ ] I know that if I don't provide sample code that allows someone to quickly step into my shoes, I may not get the help I want or my issue may be closed.

```python
import eel

...
```

```html
<html>
   ...
</html>
```

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Comment by samuelhwilliams** [comment]
I feel quite confident saying this isn't a bug - have you done `pip install eel` for Python 3.8.5? Are you using a virtual environment that doesn't have Eel installed?

**Comment by felini69** [comment]
> I feel quite confident saying this isn't a bug - have you done `pip install eel` for Python 3.8.5? Are you using a virtual environment that doesn't have Eel installed?

@samuelhwilliams thx for you answer.
I instaled last version of eel. Eel 0.14.0. 
Is there any other version for Python 3.8.5?
![image](https://user-images.githubusercontent.com/73040536/96390063-d680eb00-11c3-11eb-8138-e89f27e3cf35.png)

**Comment by samuelhwilliams** [comment]
No, that's the correct version. What I think is happening is that you have installed Eel into a different Python interpreter/environment than the one you're trying to `import eel` from. That's really the only explanation right now that springs to mind.

What happens if you run the following in a terminal:

```
pip install eel
python -c 'import eel; print(eel)'
```

**Comment by felini69** [comment]
> No, that's the correct version. What I think is happening is that you have installed Eel into a different Python interpreter/environment than the one you're trying to `import eel` from. That's really the only explanation right now that springs to mind.
> 
> What happens if you run the following in a terminal:
> 
> ```
> pip install eel
> python -c 'import eel; print(eel)'
> ```

you mean like this? or enything else?
![image](https://user-images.githubusercontent.com/73040536/96513104-a3496500-1272-11eb-9b8b-5fa4e92c9676.png)

**Comment by samuelhwilliams** [comment]
Looks like you may need to use double quotes rather than the single quotes I provided, but yes, essentially that's what I mean.

**Comment by aprilahijriyan** [comment]
check if pip is for python 3. use the option "-V" e.g. "pip -V"

**Comment by felini69** [comment]
> check if pip is for python 3. use the option "-V" e.g. "pip -V"

here is screen
![image](https://user-images.githubusercontent.com/73040536/96532338-bec96580-129c-11eb-8baf-008a874e4298.png)

**Comment by felini69** [comment]
> Looks like you may need to use double quotes rather than the single quotes I provided, but yes, essentially that's what I mean.

here is creen 
![image](https://user-images.githubusercontent.com/73040536/96532529-254e8380-129d-11eb-8a80-3d40f36825a0.png)

**Comment by aprilahijriyan** [comment]
try `python main.py` ?

**Comment by felini69** [comment]
> try `python main.py` ?

I try. It's not work on CMD and in jupyter. but vork in python IDE ((
how can I fix this ishue?
![image](https://user-images.githubusercontent.com/73040536/96656637-48d30600-1351-11eb-9f37-63b65bc633e0.png)

**Comment by felini69** [comment]
> Looks like you may need to use double quotes rather than the single quotes I provided, but yes, essentially that's what I mean.

It's not work in CMD and Jupyter but work in Python IDE ((
can we fix this ishue?
![image](https://user-images.githubusercontent.com/73040536/96656701-715b0000-1351-11eb-8c36-c8a7e1ab9e75.png)

**Comment by samuelhwilliams** [comment]
Hi @felini69 - it's good that you can see Eel is working on your machine. This means the Python interpreter you're using in Jupyter is different from the one you've installed Eel in.

You will need to find the Python interpreter that Jupyter is using and install Eel in that. I can only really point you at Jupyter docs for that problem as I'm not so familiar. This isn't an Eel issue I'm afraid.
